celerdata/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "790d8932-0833-43ac-b9d8-d6d0a4f11517",
+  "app_id": "celerdata",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gathers CelerData metrics and logs",
+    "title": "CelerData",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
```

Comment:
To confirm, the integration only supports Linux? Additionally, please include the associated Offering:: as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)

Reply:
yes, only support linux.

Reply:
done